### DISCUSSION TITLE
Help with processing MODIS Burned Area product

### DISCUSSION THREAD

**Discussion by francobarrionuevoenv21** [google/Xee#92] [Q&A] [answered]
Hi, Community! I'm very excited about this great tool, 

I'm trying to process MODIS burned area product. I'm loading the product, as it's explained in the tutorial, and I'm getting the following output

![image](https://github.com/google/Xee/assets/78272951/5769e54f-8471-4faa-9c4b-1181a574828d)

I have two questions about this:

1) The spatial resolution of this product is 500 m, so how is possible that I get a difference between points of 0.5°?
2) I noticed that the lon is upper than lat axis. I don't have much experience working with xarray, but as could compare with other datasets I've worked, it's in the opposite order: first lat and then lon coordinate. Is it possible that the following outputs is the result of this issue?

![image](https://github.com/google/Xee/assets/78272951/57377eb8-e17d-4242-b09b-0883a7beb851)

Thank you so much!!

**Comment by netherfoam** [comment, marked as answer]
> I noticed that the lon is upper than lat axis. I don't have much experience working with xarray, but as could compare with other datasets I've worked, it's in the opposite order: first lat and then lon coordinate.

Do you just need to `transpose(..)` the array? `ds.transpose("time", "lat", "lon")`, though that shouldn't affect your output at all, just the order of coordinates

> The spatial resolution of this product is 500 m, so how is possible that I get a difference between points of 0.5°?

Looks like they're 1° increments in your screenshot. EE specifies the default scale of 1 degree increments [here](https://developers.google.com/earth-engine/guides/projections). Seems the `scale=xx` parameter lets you define this when opening the dataset.

I imagine there's some way of specifying `crs`, which changes the units that `scale` uses. So surely something like mercator (which is in meters) could be used and then scale=500, but I am really struggling with projections in xee and this particular dataset myself.

**Reply by francobarrionuevoenv21** [reply]
Hi!! Thank you so much for your reply. I finally used the same product from Copernicus repository. After I finish my current work, I'll try this approach to check if this solves my issues.

**Reply by netherfoam** [reply]
Neat, looks like Copernicus has a bit higher resolution anyway, I'll have to check it out
